setup.py

```diff
@@ -8,13 +8,13 @@
 ]
 
 setup(name='jupiterone',
-      version='0.1.0',
+      version='0.3.0',
```

Comment:
Are there any breaking changes? I feel like there are. Can you confirm?

Reply:
bumped to 1.0.0